stanford/roles/cluster/defaults/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+CLUSTER_AMI: ami-73f7da13  # 16.04
+CLUSTER_AMI: ami-89f3dee9  # 14.04
```

Comment:
Why two CLUSTER_AMI variables?

Reply:
At the time, we were juggling between both 16.04 and 14.04 builds.
Now, we're 16.04 only.

I can remove the 2nd var now.

Reply:
I removed the var and updated the AMI to a more recent 16.04